src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -1083,6 +1088,89 @@ class spell_warr_victory_rush : public SpellScript
     }
 };
 
+// 228920 - Ravager
+class spell_warr_ravager : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_WARRIOR_RAVAGER_SUMMON });
+    }
+
+    void SummonRavager(SpellEffIndex /*effIndex*/) const
+    {
+        GetCaster()->CastSpell(*GetHitDest(), SPELL_WARRIOR_RAVAGER_SUMMON, CastSpellExtraArgsInit{
```

Comment:
it's best to avoid doing *DestGetter, there is a discussion in Stackoverflow where precision is always lost, it's always best to use GetHitDest()->GetPosition etc